dev/RadialGradientBrush/RadialGradientBrush.cpp

```diff
@@ -230,6 +239,29 @@ void RadialGradientBrush::UpdateCompositionInterpolationSpace()
     }
 }
 
+void RadialGradientBrush::UpdateCompositionExtendMode()
+{
+    MUX_ASSERT(SharedHelpers::IsCompositionRadialGradientBrushAvailable());
+
+    if (const auto compositionGradientBrush = m_brush.try_as<winrt::CompositionRadialGradientBrush>())
+    {
+        switch (SpreadMethod())
+        {
+        case winrt::GradientSpreadMethod::Repeat:
+            compositionGradientBrush.ExtendMode(winrt::Windows::UI::Composition::CompositionGradientExtendMode::Wrap);
+            break;
+        case winrt::GradientSpreadMethod::Reflect:
+            compositionGradientBrush.ExtendMode(winrt::Windows::UI::Composition::CompositionGradientExtendMode::Mirror);
+            break;
+        case winrt::GradientSpreadMethod::Pad:
+            [[fallthrough]];
```

Comment:
>[[fallthrough]]; [](start = 12, length = 16)

cpp guideliens say explicit fallthroughs should be rare but when deemed appropriate are okay. However they also say multiple case labels of a single statement are okay.  I think you could delete this,  and maybe should? I'm curious what other reviewers might think.

[https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es78-dont-rely-on-implicit-fallthrough-in-switch-statements](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es78-dont-rely-on-implicit-fallthrough-in-switch-statements)

Reply:
I think the issue is with implicit fallthrough. I think I am find with either merging the two or the explicit way. In both cases the intention is clear.

---
In reply to: [372529304](https://github.com/microsoft/microsoft-ui-xaml/pull/1903#discussion_r372529304) [](ancestors = 372529304)